pkg/v1/providers/ytt/02_addons/csi/csi_addon_data.lib.yaml

```diff
@@ -56,6 +76,9 @@ vsphereCSI:
   http_proxy: #@ data.values.TKG_HTTP_PROXY
   https_proxy: #@ data.values.TKG_HTTPS_PROXY
   no_proxy: #@ get_no_proxy()
+  #@ if enableTopologyFeature():
+  useTopologyFeature: True
```

Comment:
Suggestion: will `useTopologyFeature : #@ enableTopologyFeature()` be a good idea ?

Reply:
Right, will change this line.